extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -52,10 +58,22 @@
     @ConfigRoot(name = "smallrye-health")
     static final class SmallRyeHealthConfig {
         /**
-         * The path of the health-checking servlet.
+         * The path of the legacy health-checking servlet.
          */
         @ConfigItem(defaultValue = "/health")
-        String path;
+        String healthPath;
```

Comment:
Not sure if this is possible, but is the `healthPath` the root for the `livenessPath` and `readinessPath`?

Or do we expect a user to be able to change `/health` to `/bubbles` and not impact the other two?

Reply:
+1 for having `healthPath` as a root for live and ready.